src/playwrightTestServer.ts

```diff
@@ -91,7 +91,7 @@ export class PlaywrightTestServer {
     if (!connection)
       return;
     // Locations are regular expressions.
-    locations = locations.map(escapeRegex);
+    locations = locations.map(preventRegexLookalike).map(escapeRegex);
```

Comment:
Since we call `escapeRegex` - perhaps we don't need `preventRegexLookalike` here at all?

Reply:
`escapeRegex` doesn't do anything to forward slashes. I can merge the `preventRegexLookalike` logic into `escapeRegex` if you'd prefer that.

Reply:
Yep, that would be great. Just escape the forward slashes together with other characters.

Reply:
Escaping the forward slashes wasn't enough, but removing the leading one did the trick: https://github.com/microsoft/playwright-vscode/pull/614/commits/42995e8ae458ffd23f11aa5b69fc1637d2d60f74

I also made sure that the exec log always shows the right trailing slash. `path.relative` likes to remove that.